### PR TITLE
Layer5 ng

### LINKED ISSUE #1316: [NG] Community: Recognizing Adekanye Tomi as a Contributor and Community Member
**Current Behavior**

@adekanyetomie has been a consistent contributor and community member.

**Desired Behavior**
A profile for @adekanyetomie should be created and visible on /community/members on next generation layer5.io site.

 Github: https://github.com/adekanyetomie
 Twitter: adetomiie
 LinkedIn: oluwatomiyin-adekanye-485852150
 Profile picture: https://twitter.com/adetomiie/photo

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Elijah699**
**Description**

This PR fixes #1316 and a typographical error in Olushesi Toheeb index.mdx file

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/oluwatomiyin-adekanye/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+github: adekanyetomie
+twitter: adetomiie
+linkedin: oluwatomiyin-adekanye-485852150
+blurb: Mesh Head
+status: Active
+badges:
+  - community member
+location: Nigeria
+bio: I am a creative software engineer focused on implementing scalable solutions, thorough communication, writing efficient code, and collaboration with people. I have strong interest in open-source development, advocacy and community building.
+published: false
```
Comment (Jashpatel1) [inline src/collections/members/oluwatomiyin-adekanye/index.mdx:14]: Change it to true. Everything else looks fine. @adekanyetomie

**Comment by leecalcote** [comment]
@Elijah699 will you take a moment to add a more descriptive title to this PR?

Yay. @adekanyetomie! Please review.

**Comment by adekanyetomie** [comment]
@leecalcote it's perfect

**Review by Tanuj22 — APPROVED** [review]
Thank you @Elijah699 and @adekanyetomie

**Pull request merged**